apix/config/v1alpha1/endpointpickerconfig_types.go

```diff
@@ -118,3 +130,68 @@ func (sp SchedulingPlugin) String() string {
 	}
 	return fmt.Sprintf("{PluginRef: %s%s}", sp.PluginRef, weight)
 }
+
+// FeatureGates is a set of flags that enable various experimental features with the EPP
+type FeatureGates struct {
+	// +optional
+	// EnableDataLayer If present and true enables the experimental Datalayer APIs.
+	// The Datalayer APIswill be disabled if EnableDataLayer is not present in the
+	// configuration or its value is false.
+	EnableDataLayer bool `json:"enableDataLayer,omitempty"`
+
+	// +optional
+	// EnableFlowControl If present and true enables the experimental FlowControl feature.
+	// The FlowControl feature will be disabled if EnableFlowControl is not in the
+	// configuration or its value is false.
+	EnableFlowControl bool `json:"enableFlowControl,omitempty"`
+}
```

Comment:
feature gates by definition are boolean to enable/disable a feature. 
can we just call it `DataLayer` and `FlowControl`?